### PR TITLE
test latest conda-build version

### PR DESCRIPTION
**Pull request by bgruening**

### CODE REVIEW

**Comment by nsoranzo** [comment]
The second error in the Travis build is due to this change: https://github.com/conda/conda-build/commit/2fcded8c67259ad3c2cd5b5f4acdc8241a1c9122#diff-a273b977a04f704063ec213a6230e203R68

Changing:
```
print(bldpkg_path(metadata, metadata.config))
```
to
```
print(bldpkg_path(metadata))
```
at line 467 of `test/test_utils.py` should fix it.

File: test/test_utils.py [hunk 1]
```diff
@@ -464,7 +464,7 @@ def test_filter_recipes_extra_in_build_string():
     from conda_build.render import bldpkg_path
 
     metadata = MetaData(recipe, api.Config(**dict(CONDA_EXTRA='asdf')))
-    print(bldpkg_path(metadata, metadata.config))
+    print(bldpkg_path(metadata))
```
Comment (johanneskoester) [inline test/test_utils.py]: Maybe it would be better to use API.get_output_file_path as we use it for filtering recipes.

Comment (bgruening) [inline test/test_utils.py]: You mean `api.get_output_file_path(recipe, metadata.config)`

Comment (johanneskoester) [inline test/test_utils.py]: Yes, I think so.

Comment (johanneskoester) [inline test/test_utils.py]: Just to be sure that we capture the same problem as we see it when filtering.

**Comment by johanneskoester** [comment]
Seems like it finally works!

**Comment by bgruening** [comment]
Cool, let's see if this fixes the bulk branch.

**Comment by johanneskoester** [comment]
Make sure that you merge master into bulk before starting a rebuild. Otherwise, it might rebuild the duplicate recipes again that we are now deleting from anaconda.

**Pull request merged**